ScoutSuite/frontend/src/api/useAPI.js

```diff
@@ -12,14 +12,16 @@ export const useAPI = (defaultPath, defaultValue, settings = {}) => {
   );
   const [path, setPath] = useState(defaultPath);
   const [loading, setLoading] = useState(!Cache.has(path));
+  const [reloading, setReloading]= useState(false);
```

Comment:
what's the use for a reloading state?

Reply:
When loading a new page in the tables, we do not want the table to "flash". Reloading state tells that new content is being fetched but the page should should a loading state (or could show a loader but the content should stay interactive).